packages/image-block/src/settings.ts

```diff
@@ -1,28 +1,299 @@
+                                    value: BorderStyle.Dotted,
+                                    label: 'Dotted',
+                                },
+                                {
+                                    value: BorderStyle.Dashed,
+                                    label: 'Dashed',
+                                },
+                                {
+                                    value: BorderStyle.Solid,
+                                    label: 'Solid',
+                                },
+                            ],
+                        },
+                        {
+                            id: 'borderWidth',
+                            type: 'input',
+                            defaultValue: '1px',
+                            placeholder: 'e.g. 2px',
+                            rules: [numericalOrPixelRule],
+                            clearable: false,
+                        },
+                        {
+                            id: 'borderColor',
+                            type: 'colorInput',
+                            defaultValue: { r: 153, g: 153, b: 153 },
+                        },
+                    ],
+                    layout: MultiInputLayout.Columns,
+                },
+            ],
+        },
+        {
+            id: 'cornerRadius',
+            type: 'switch',
+            defaultValue: false,
+            switchLabel: 'Custom',
+            label: 'Corner radius',
+            onChange: (bundle) => presetCustomValue(bundle, RADIUS_CHOICE_ID, RADIUS_CUSTOM_ID, radiusValues),
+            on: [
+                {
+                    id: RADIUS_CUSTOM_ID,
+                    type: 'input',
+                    rules: [numericalOrPixelRule],
+                    onChange: (bundle) => appendUnit(bundle, RADIUS_CUSTOM_ID),
+                },
+            ],
+            off: [
+                {
+                    id: RADIUS_CHOICE_ID,
+                    type: 'slider',
+                    defaultValue: Padding.None,
+                    choices: [
+                        {
+                            value: Padding.None,
+                            label: 'None',
+                        },
+                        {
+                            value: Padding.Small,
+                            label: 'S',
+                        },
+                        {
+                            value: Padding.Medium,
+                            label: 'M',
+                        },
+                        {
+                            value: Padding.Large,
+                            label: 'L',
+                        },
+                    ],
+                },
+            ],
+        },
+    ],
+    security: [
+        {
+            id: 'security',
+            type: 'slider',
+            defaultValue: ImageSecurity.Global,
+            info: 'This setting is only available for images that are uploaded to Frontify.',
```

Comment:
In the design there is a comment `Change global settings here.`. Do you know about this?

The info `This setting is only available for images that are uploaded to Frontify.` is not shown. Probably because there is no label?